website/blog/2022-08-22-narrative-modeling.md

```diff
@@ -0,0 +1,236 @@
+- After cleaning up source data with staging tables, use the same “what happened” approach to more technical events, creating a three-node dependency from `stg_snowplow_events` to  `int_page_click_captured` to `user_refreshed_cart`  and thus answering the question “where do we get online user behavior information?” in a quick visit to the DAG in dbt docs.
+
+# Should your team use it?
+
+Narrative modeling primarily values comprehension as an outcome of data modeling, which can be a high priority for…
+
+- …companies with a high stakeholder-to-data-team-member ratio.
+- …companies with SQL-savvy stakeholders
+- …companies looking to on-board new team members at a high rate (as the project in this case is a sketch of the business itself)
+- …companies who can dedicate staffing and time to writing strong documentation so that the doors to the models can be thrown open.
+
+Narrative modeling might not be the right fit for…
+
+- …companies where storage spend, even in cloud warehouses, needs to be closely tracked. After all, fct/dim models emerged from a need to optimize data storage.
+- …companies with BI tools that rely heavily on multi-entity tables. This can generally still work with narrative modeling, however, if there is an additional layer of standard datasets being modeled out of the shared components of your narrative models (to insure data stays in step across contexts).
+
+# A neverending story
+
+There’s a classic movie called Desk Set in which Katharine Hepburn runs the reference desk of a major TV network and Spencer Tracy comes in to install a research computer in her department. In gladiatorial bouts of banter, the movie investigates the concept of knowledge and, in particular, how human it should be. At one point, Tracy’s Richard Sumner runs Hepburn’s Bunny Watson through a chunky “train left the station” word problem and I think about her response (abbreviated below) all the time. 
+
+**Richard Sumner:** Now, a train started out at Grand Central, with seventeen passengers aboard and a crew of nine. At 125th Street, four got off and nine got on. At White Plains, three got off and one got on. At Chappaqua, nine got off and four got on. And at each successive stop thereafter, nobody got off, nobody got on till the train reached its next-to-the-last stop, where five people got off and one got on. Then it reached the terminal.
+
+**Richard Sumner:** How many people got off at Chappaqua?
+
+**Bunny Watson:** Nine.
+
+**Richard Sumner:** Uh, would you mind telling me how you arrived at that conclusion?
+
+**Bunny Watson:** Spooky, isn't it? Did you notice that there are also nine letters in Chappaqua?
+
+**Richard Sumner:** Are you in the habit of associating words with the number of letters in them?
+
+**Bunny Watson:** I associate many things with many things.
+
+Richard Sumner: I see. Hmm.
```

Comment:
all of Richard/Bunny are un bolded after this line. Is that on purpose?

Reply:
Not intentional. Fixed!